visualizer/src/Controls/ImageControls/LabeledControls/OutlineToggle.js

```diff
@@ -0,0 +1,47 @@
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormGroup from '@material-ui/core/FormGroup';
+import Switch from '@material-ui/core/Switch';
+import Tooltip from '@material-ui/core/Tooltip';
+import { useSelector } from '@xstate/react';
+import React, { useEffect, useRef } from 'react';
+import { useLabeled } from '../../../ServiceContext';
+
+function OutlineToggle() {
+  const labeled = useLabeled();
+  const outline = useSelector(labeled, state => state.context.outline);
+  const { send } = labeled;
+
+  // Adds mousetrap class so hotkeys work after using switch
+  const inputRef = useRef();
+  useEffect(() => {
+    const input = inputRef.current;
+    input.className = input.className + ' mousetrap';
```

Comment:
Prefer the string formatting over concatenation:

```suggestion
    input.className = `${input.className}  mousetrap`;
```